### PR TITLE
Added clear project requirements for week 8 game.

### PR DESCRIPTION
**Pull request by Lakhwinderr**
Added new project requirements for users to customize the trivia game by selecting the number of questions, question type, difficulty, and category.

### CODE REVIEW

File: projects/week8GameREADME.md [hunk 1]
```diff
@@ -1,65 +1,112 @@
...
+### Step-by-Step Development Plan
 
-- Make the game functional
+1. **Frontend Initialization**:
 
-## Guide code
+   - Install dependencies and set up React with Vite.
 
-You can see a tutorial on how to create a game in React [here](https://www.freecodecamp.org/news/how-to-build-a-quiz-app-using-react/). Please note that this tutorial uses `create-react-app` as the initial framework for the Frontend, and you don't want to use it. 
+   - Create initial components: GameSetup, GamePlay, and GameResult.
```
Comment (daaimah123) [inline projects/week8GameREADME.md:79]:
```suggestion
   - Create initial components such as `GameSetup`, `GamePlay`, and `GameResult`.
```

File: projects/week8GameREADME.md [hunk 2]
```diff
@@ -1,65 +1,112 @@
...
+5. Integrating frontend and backend to fetch and display external API data.
 
-This assignment will check for proficiency in NodeJS and ExpressJS and how to connect your Backend with the frontend.
+## Game Overview
 
-### Overview
+This game fetches trivia questions from an external API, displays them on the frontend, and allows users to answer them. At the end of the game, the backend determines if the user has won or lost based on their answers.
 
-In this project, you will build a minimal Game connecting your Backend and your frontend. 
+The game's existing code already includes functionality to fetch user information and display questions from the `fakedata.js` file. Your task will be to replace this with real API data and implement additional features.
```
Comment (daaimah123) [inline projects/week8GameREADME.md:30]:
```suggestion
The game's existing code already includes functionality to fetch user information and display questions from the `fakedata.js` file. Your task will be to replace this with real API data and implement additional features. See [example game code](https://github.com/Yosolita1978/week8Game).
```

File: projects/week8GameREADME.md [hunk 3]
```diff
@@ -1,65 +1,112 @@
...
-It should grab how many questions the user has an answer to and how many right/wrong questions.
+1. Create a full-stack project structure:
+   - In the root directory, create a `server` folder for the Express server.
+   - Using Vite, create a `client` folder for the React frontend. Follow [this guide](https://github.com/Techtonica/curriculum/blob/main/react-js/react-local-vite.md) for setup.
 
-##### Step #3
+### Backend Instructions
 
-It should show the user a message in case they win/lose the game.
+1. **API Request Setup**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:53]:
```suggestion
- [ ] **API Request Setup**:
```

File: projects/week8GameREADME.md [hunk 4]
```diff
@@ -1,65 +1,112 @@
...
+   - Set up a GET route in Express to fetch trivia questions from the external API (e.g., [Open Trivia Database](https://opentdb.com/api_config.php)).
+   - Configure the API request based on the user’s customization settings (number of questions, type, difficulty, category).
+   - Ensure that API responses are properly formatted and accessible to the frontend.
 
+### Frontend Instructions
 
-# Project Instructions
+1. **Game Setup Screen**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:60]:
```suggestion
- [ ] **Game Setup Screen**:
```

File: projects/week8GameREADME.md [hunk 5]
```diff
@@ -1,65 +1,112 @@
...
 
-## Part 0 - Starter Code
+   - Create a setup screen where users can select their preferences (number of questions, type, difficulty, category).
+   - Use form controls to capture and manage the user’s input.
 
-- [ ] Start by starting a full-stack project from scratch - in your root directory for your project, create a server directory and start an express server in it.  Using Vite, create a client directory [This lesson covers the front-end part if you want to review it](https://github.com/Techtonica/curriculum/blob/main/react-js/react-local-vite.md)
+2. **Game Interface**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:65]:
```suggestion
- [ ] **Game Interface**:
```

File: projects/week8GameREADME.md [hunk 6]
```diff
@@ -1,65 +1,112 @@
...
 
-### For the Backend
+   - Display questions based on the user’s settings, fetched from the backend.
+   - Track the user’s answers and keep a count of correct and incorrect answers.
 
-- Make the GET request for real data to the API in the Backend
+3. **Result Display**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:70]:
```suggestion
- [ ] **Result Display**:
```

File: projects/week8GameREADME.md [hunk 7]
```diff
@@ -1,65 +1,112 @@
...
+   - Show a final message indicating whether the user won or lost based on their score.
 
-### For the Frontend
+### Step-by-Step Development Plan
 
-- Make the game functional
+1. **Frontend Initialization**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:75]:
```suggestion
- [ ] **Frontend Initialization**:
```

File: projects/week8GameREADME.md [hunk 8]
```diff
@@ -1,65 +1,112 @@
...
 
-## Guide code
+   - Install dependencies and set up React with Vite.
 
-You can see a tutorial on how to create a game in React [here](https://www.freecodecamp.org/news/how-to-build-a-quiz-app-using-react/). Please note that this tutorial uses `create-react-app` as the initial framework for the Frontend, and you don't want to use it. 
+   - Create initial components: GameSetup, GamePlay, and GameResult.
+
+2. **Backend Initialization**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:81]:
```suggestion
- [ ] **Backend Initialization**:
```

File: projects/week8GameREADME.md [hunk 9]
```diff
@@ -1,65 +1,112 @@
...
+
+   - Install dependencies and set up Express in the `server` directory.
+
+   - Set up environment configuration for API keys (if needed).
+   - Create an endpoint to handle API requests based on customization parameters.
+
+3. **Frontend-Backend Integration**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:88]:
```suggestion
- [ ] **Frontend-Backend Integration**:
```

File: projects/week8GameREADME.md [hunk 10]
```diff
@@ -1,65 +1,112 @@
...
+
+   - Use the Fetch API or Axios to send user preferences from the frontend to the backend.
+   - Display API-fetched questions on the frontend and handle user interactions.
+
+4. **Win/Loss Logic**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:93]:
```suggestion
- [ ] **Win/Loss Logic**:
```

File: projects/week8GameREADME.md [hunk 11]
```diff
@@ -1,65 +1,112 @@
...
+
+   - Track the user's responses and send the score to the backend.
+   - The backend determines if the user has won or lost, returning the result to the frontend.
+
+5. **Testing and Debugging**:
```
Comment (daaimah123) [inline projects/week8GameREADME.md:98]:
```suggestion
- [ ] **Bonus Feature: Testing and Debugging**:
```

**Review by daaimah123 — CHANGES_REQUESTED** [review]

**Comment by Lakhwinderr** [comment]
Thanks 😊 @daaimah123

**Comment by daaimah123** [comment]
@Lakhwinderr "hacktoberfest-accepted"

**Pull request merged**